src/current/_includes/metrics-usage/tenant.consumption.external_io_egress_bytes.md

```diff
@@ -0,0 +1,5 @@
+The number of RUs that were consumed by byte traffic to cluster bulk I/O operations (e.g., CDC). Egress bytes are converted to Request Units using this equivalency:
+<br>
+<b>1 RU = 1 KiB Network egress.</b>
+<br>
+Correlate this metric with Request Units (RUs). <a href="https://www.cockroachlabs.com/docs/cockroachcloud/serverless-resource-usage">Learn more</a>.
```

Comment:
the description of this link should be changed from "learn more" to something more descriptive (per [style guide](https://github.com/cockroachdb/docs/blob/main/StyleGuide.md#links))

> Avoid using non-descriptive link names such as here, this page, or go.

suggest something like 'to learn more about how RUs are calculated, see $PAGE' (whatever page title is)